src/debugger/channel/MessageMarshaller.js

```diff
@@ -43,13 +44,15 @@ export class MessageMarshaller {
     return `${requestID} ${messageType} ${JSON.stringify(breakpoints)}`;
   }
 
-  marshallBreakpointStopped(args: Breakpoint): string {
-    return `${this
-      ._lastRunRequestID} ${DebugMessage.BREAKPOINT_STOPPED_RESPONSE} ${args.filePath} ${args.line} ${args.column}`;
-  }
-
-  marshallPrepackFinish(): string {
-    return `${this._lastRunRequestID} ${DebugMessage.PREPACK_FINISH_RESPONSE}`;
+  marshallPrepackStopped(reason: PrepackStoppedReason, filePath: string, line: number, column: number): string {
```

Comment:
It's weird to have "Prepack" in this. Let's call it, marshallStoppedResponse or marshallStoppedEvent